spec/features/account_creation/completions_cancel_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.feature 'asks users if they want to exit to partner agency', allowed_extra_analytics: [:*] do
+  include SamlAuthHelper
+
+  it 'redirects accordingly' do
+    visit_idp_from_sp_with_ial1(:oidc)
+    sign_up_and_set_password
+    select_2fa_option('backup_code')
+    click_continue
+
+    expect(current_path).to eq('/sign_up/completed')
+
+    click_cancel
```

Comment:
1. As caught in the failure, there's not a helper for `click_cancel`. We can click on the text instead.
2. Can we make sure we assert something about what's expected to be seen on the cancellation screen? And even better to assert what happens after clicking the next options.

```suggestion
    click_on t('links.cancel')

    expect(current_path).to eq(sign_up_completed_cancel_path)
    click_on t('login_cancel.keep_going')

    expect(current_path).to eq(sign_up_completed_path)
    click_on t('links.cancel')

    expect(current_path).to eq(sign_up_completed_cancel_path)
    click_on t('login_cancel.exit', app_name: APP_NAME)

    expect(current_url).to start_with('http://localhost:7654/auth/result?error=access_denied')
```

Reply:
> 1. As caught in the failure, there's not a helper for `click_cancel`. We can click on the text instead.
> 2. Can we make sure we assert something about what's expected to be seen on the cancellation screen? And even better to assert what happens after clicking the next options.

1. I was messing around with creating a `click_cancel` helper. It was not worth the work in this ticket. I can do that work in another PR.

2. Yes, this is the direction that I am working on. I also want to assert that it goes to the previous page.